### ISSUE TITLE
Use alt text to describe the image

### ISSUE THREAD

**Issue by discospiff** [aalonge2/IS-7012-WEB-DEV-GitHub#5]
Same comment as before...

Use alt text to give an accurate, yet concise description of what the image looks like, for the benefit of users who cannot see the image.

https://github.com/aalonge2/IS-7012-WEB-DEV-GitHub/blob/9c3581bb212818d131ef7b3dad8ce3d1ce99982b/RecruitCatLongeaa/Pages/OurTeam.cshtml#L22
